ProcessSlave.php

```diff
@@ -384,18 +391,20 @@ protected function handleRequest(\React\Http\Request $request, HttpResponse $res
             );
             $this->shutdown();
         }
+        return $response;
     }
 
-    protected function prepareEnvironment(\React\Http\Request $request)
+    protected function prepareEnvironment(RequestInterface $request)
     {
         $_SERVER = $this->baseServer;
```

Comment:
No, sorry, You misunderstood me. "this is bad" was NOT about Your comment, but about populating SERVER vars ;) So I totally agree with You about removing this SERVER code. IMO, populating SERVER vars may be implemented with ServerRequestInterface (in `react/http`) out of the box, so no need our code for it.